packages/components/src/FormatFile/FormatFile.css

```diff
@@ -69,10 +133,24 @@
   text-overflow: ellipsis;
   white-space: nowrap;
 }
+
 .contentBlock p {
   margin: 0;
 }
 
 .actionBlock {
   padding: 0 var(--file-card--base-padding) 0 0;
 }
+
+.thumbnail {
+  display: flex;
+  position: relative;
+}
+
+.deleteButton {
+  display: flex;
+  position: absolute;
+  top: var(--space-smaller);
+  right: var(--space-smaller);
+  z-index: 1;
```

Comment:
Seeing a lot of z-index usage in this component, is it possible to avoid it by ordering the DOM elements in a natural stacking order? It looks like this isn't even needed when I remove the style:
![image](https://user-images.githubusercontent.com/39704901/171214870-2a45409b-9962-404a-811d-478db8209c68.png)